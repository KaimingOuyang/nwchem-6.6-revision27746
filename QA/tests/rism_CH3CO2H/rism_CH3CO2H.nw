echo
start rism

memory global 40 mb stack 23 mb heap 5 mb

rism
 closure kh
 tol 0.1e-6
 temp 298
 vdw rule arithmetic parameters vdw.par
 solute configuration solute2.data
 ngrid 4096
end
task energy rism

