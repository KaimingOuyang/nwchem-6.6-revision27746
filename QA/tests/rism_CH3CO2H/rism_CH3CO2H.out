 argument  1 = h2o.nw
 


============================== echo of input deck ==============================
echo
start rism

memory global 40 mb stack 23 mb heap 5 mb

rism
 closure kh
 tol 0.1e-6
 temp 298
 vdw rule arithmetic parameters vdw.par
 solute configuration solute2.data
 ngrid 4096
end
task energy rism

================================================================================


                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.6
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2015
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = g0
    program         = /home/marat/codes/nwchem-6.6-old/bin/LINUX64/nwchem
    date            = Mon Oct 19 14:04:56 2015

    compiled        = Mon_Oct_19_13:52:21_2015
    source          = /home/marat/codes/nwchem-6.6
    nwchem branch   = 6.6
    nwchem revision = 27722
    ga revision     = 10594
    input           = h2o.nw
    prefix          = rism.
    data base       = ./rism.db
    status          = startup
    nproc           =        2
    time left       =    351s



           Memory information
           ------------------

    heap     =     655356 doubles =      5.0 Mbytes
    stack    =    3014657 doubles =     23.0 Mbytes
    global   =    5242880 doubles =     40.0 Mbytes (distinct from heap & stack)
    total    =    8912893 doubles =     68.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 
 ****************************************************
 *   NWChem 1D-RISM Module                          *
 *                                                  *
 *   Marat Valiev and Gennady N. Chuev 2011         *
 ****************************************************
                  1D-RISM Calculation Parameters
                  ------------------------------
Number of grid points           4096
Closure type                    kh              
VDW combination rule            arithmetic      
Solvent permittivity             1.000    
Coulomb decay length             1.000    
Convergence tolerance           0.1000E-06
Mixing parameter                 0.000    
Temperature                      298.0    
Diis iterations                    9
Solvent rdf file                /home/marat/codes/nwchem-6.6/src/rism/data/water.rdf                    
Solute configuration file       solute2.data                                                            
                       Solute configuration
                       --------------------
name type       x           y           z           sigma       eps         charge
O1   O1         0.155667   -0.860695    1.925632    3.066000    0.880900   -0.732310
O2   O2         2.313025   -0.615505    1.328693    2.960000    0.879200   -0.772125
C1   C1         0.692523   -1.269426   -0.348149    3.400000    0.458000   -0.444420
C2   C2         1.159677   -0.885318    1.026428    3.400000    0.360100    1.004562
H1   H1         0.228620   -2.261607   -0.310111    2.115000    0.065700    0.130396
H2   H2        -0.081705   -0.566546   -0.678347    2.115000    0.065700    0.151321
H3   H3         1.531381   -1.283512   -1.046441    2.115000    0.065700    0.145452
H4   H4         0.486809   -0.663628    2.835513    0.800000    0.192600    0.517125
 starting iterations
                     2                     2   2.27706494644304     
                     3                     3   1.18491074296892     
                     4                     4   1.28508381921437     
                     5                     5   1.47803514438160     
                     6                     6   2.04096995935106     
                     7                     7   2.90933775881034     
                     8                     8   4.36022769553321     
                     9                     9   6.64165669110173     
                    10                     9   10.6163006255701     
                    11                     9  0.106770685959373     
                    12                     9  9.039278868821957E-002
                    13                     9  7.519092448593380E-002
                    14                     9  7.330942674118247E-002
                    15                     9  8.322274213266166E-002
                    16                     9  9.192960109164970E-002
                    17                     9  9.352110561733681E-002
                    18                     9  9.195477012970772E-002
                    19                     9  6.196828655298060E-002
                    20                     9  4.005781205371975E-002
                    21                     9  3.911268960862913E-002
                    22                     9  2.794162744519978E-002
                    23                     9  2.276374906009340E-002
                    24                     9  2.354854531570630E-002
                    25                     9  2.264369614129940E-002
                    26                     9  2.135074541854817E-002
                    27                     9  2.069615368067657E-002
                    28                     9  1.894504344346812E-002
                    29                     9  1.985356276217899E-002
                    30                     9  1.473288893904707E-002
                    31                     9  1.371096389779174E-002
                    32                     9  1.314208452775447E-002
                    33                     9  1.285905515575548E-002
                    34                     9  1.314307191994220E-002
                    35                     9  1.356143930659721E-002
                    36                     9  1.393291624515391E-002
                    37                     9  1.433173135928520E-002
                    38                     9  1.429097558924706E-002
                    39                     9  1.355097223695083E-002
                    40                     9  1.266814964599350E-002
                    41                     9  1.144582107742443E-002
                    42                     9  1.219393567748105E-002
                    43                     9  1.172827006987542E-002
                    44                     9  9.907749660785127E-003
                    45                     9  7.913279582106230E-003
                    46                     9  6.541692124357915E-003
                    47                     9  5.427348098861998E-003
                    48                     9  6.357337538059041E-003
                    49                     9  5.664930243389963E-003
                    50                     9  5.107833078489481E-003
                    51                     9  4.776572618001453E-003
                    52                     9  4.732043736927849E-003
                    53                     9  4.679238230170962E-003
                    54                     9  4.343623936389893E-003
                    55                     9  4.526040450448660E-003
                    56                     9  4.575463052016480E-003
                    57                     9  4.127576644026400E-003
                    58                     9  4.220172319274335E-003
                    59                     9  4.143795955575791E-003
                    60                     9  3.986059544687896E-003
                    61                     9  3.944882778169836E-003
                    62                     9  3.962263337403581E-003
                    63                     9  4.010722523506454E-003
                    64                     9  4.095123776213608E-003
                    65                     9  4.181984637552710E-003
                    66                     9  4.024455188357493E-003
                    67                     9  3.891387005745420E-003
                    68                     9  4.480030387710236E-003
                    69                     9  4.101787328105021E-003
                    70                     9  4.285076463240111E-003
                    71                     9  4.262580705346527E-003
                    72                     9  4.336758922659634E-003
                    73                     9  4.408898618478129E-003
                    74                     9  4.448832312421807E-003
                    75                     9  4.617468386399179E-003
                    76                     9  4.402014252239147E-003
                    77                     9  3.246494383866174E-003
                    78                     9  4.263592220562678E-003
                    79                     9  3.911500742185026E-003
                    80                     9  4.311403468965533E-003
                    81                     9  4.234263340224797E-003
                    82                     9  4.313611137537704E-003
                    83                     9  4.454128968101511E-003
                    84                     9  4.465310763249874E-003
                    85                     9  3.839495300169527E-003
                    86                     9  3.747311168541275E-003
                    87                     9  3.186519679844505E-003
                    88                     9  3.746810954751696E-003
                    89                     9  3.025552062746893E-003
                    90                     9  2.569319881788581E-003
                    91                     9  2.454589698933495E-003
                    92                     9  2.255200648629271E-003
                    93                     9  2.181831605264842E-003
                    94                     9  2.208668330945313E-003
                    95                     9  2.095679704706112E-003
                    96                     9  1.996248183093823E-003
                    97                     9  1.662152553015151E-003
                    98                     9  1.488704721722274E-003
                    99                     9  1.283018107055400E-003
                   100                     9  1.060432850987070E-003
                   101                     9  8.383984703714915E-004
                   102                     9  8.444033788283372E-004
                   103                     9  6.599257660621085E-004
                   104                     9  5.343445147828506E-004
                   105                     9  5.159348196978628E-004
                   106                     9  4.166734896600778E-004
                   107                     9  3.359533991227795E-004
                   108                     9  3.252600154625508E-004
                   109                     9  2.997308014574322E-004
                   110                     9  2.994104677622376E-004
                   111                     9  2.564539236691421E-004
                   112                     9  2.356242708444562E-004
                   113                     9  2.147571197294054E-004
                   114                     9  1.589215631517223E-004
                   115                     9  1.283108718084492E-004
                   116                     9  1.107664368549916E-004
                   117                     9  8.961789524370540E-005
                   118                     9  7.429226822558448E-005
                   119                     9  7.442703209202344E-005
                   120                     9  6.873541126694864E-005
                   121                     9  6.914851296720845E-005
                   122                     9  6.819665011575490E-005
                   123                     9  6.824425970403713E-005
                   124                     9  6.882919822358444E-005
                   125                     9  6.778314696887985E-005
                   126                     9  6.611001905509922E-005
                   127                     9  6.074055362397134E-005
                   128                     9  5.851355056485887E-005
                   129                     9  5.862851658344722E-005
                   130                     9  5.780498780286205E-005
                   131                     9  5.649918509979568E-005
                   132                     9  5.489151609932387E-005
                   133                     9  5.379928057083481E-005
                   134                     9  5.184051019738119E-005
                   135                     9  4.074274802069194E-005
                   136                     9  2.481711295501808E-005
                   137                     9  2.077074644746927E-005
                   138                     9  1.995159494957721E-005
                   139                     9  1.901286020699157E-005
                   140                     9  1.642302664379331E-005
                   141                     9  1.041002244378211E-005
                   142                     9  6.582847622202864E-006
                   143                     9  5.575764325104635E-006
                   144                     9  5.253855799979980E-006
                   145                     9  5.130890781469311E-006
                   146                     9  4.491163999361236E-006
                   147                     9  4.263810705263765E-006
                   148                     9  3.423472416002945E-006
                   149                     9  3.346499013022716E-006
                   150                     9  3.038056829205412E-006
                   151                     9  2.850403579578217E-006
                   152                     9  2.727545165581286E-006
                   153                     9  2.367790371556875E-006
                   154                     9  2.403646455725880E-006
                   155                     9  1.823273729769791E-006
                   156                     9  1.660925238911822E-006
                   157                     9  1.397522688081430E-006
                   158                     9  1.309525028498913E-006
                   159                     9  1.215331919109254E-006
                   160                     9  1.161397023655914E-006
                   161                     9  1.037453231324435E-006
                   162                     9  7.196448927010959E-007
                   163                     9  6.819129620611833E-007
                   164                     9  5.525606626128324E-007
                   165                     9  4.601846672720351E-007
                   166                     9  3.913080077940600E-007
                   167                     9  3.615725423646456E-007
                   168                     9  3.064618111991796E-007
                   169                     9  2.585623693841061E-007
                   170                     9  2.379546963968724E-007
                   171                     9  2.217834495535622E-007
                   172                     9  1.895680438113644E-007
                   173                     9  1.575319463438367E-007
                   174                     9  1.186891711215213E-007
                   175                     9  1.124602145018037E-007
                   176                     9  1.154110569371974E-007
                   177                     9  1.136648061092910E-007
                   178                     9  1.075307796926969E-007
                   179                     9  1.074889737936231E-007
                   180                     9  1.038675346679026E-007
                   181                     9  8.452192017468949E-008
Chemical potential(hnc approximation)       -12.7987 kcal/mol
Chemical potential(gaussian approximation)  -15.0710 kcal/mol

 Task  times  cpu:        3.4s     wall:        4.7s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:    0        0        0        0        0        0        0        0     
number of processes/call 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
bytes total:             0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 0 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        42	         0
	current total bytes		         0	         0
	maximum total bytes		   2169160	         0
	maximum total K-bytes		      2170	         0
	maximum total M-bytes		         3	         0
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
       S. A. Fischer, S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen,
    V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus, M. Malagoli,
       A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati,
     J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, M. Williamson, R. J. Harrison,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
    M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
      G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall,
      J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt,
      P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall,
    D. Elwood, E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson,
     J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        3.4s     wall:        4.7s
